src/shrutiS/MathWithWhile.java

```diff
@@ -0,0 +1,58 @@
+package shrutiS;
+
+public class MathWithWhile {
+
+	void evenNumbers(int startNumber, int endNumber) {
+		System.out.println("Even Numbers are : ");
+		int evenNumber = startNumber;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use startNumber++ for increment?